stackslib/src/net/stackerdb/mod.rs

```diff
@@ -152,7 +152,7 @@ pub const STACKERDB_MAX_PAGE_COUNT: u32 = 2;
 
 pub const STACKERDB_SLOTS_FUNCTION: &str = "stackerdb-get-signer-slots";
 pub const STACKERDB_CONFIG_FUNCTION: &str = "stackerdb-get-config";
-pub const MINER_SLOT_COUNT: u32 = 2;
+pub const MINER_SLOT_COUNT: u32 = 3;
```

Comment:
I don't think you need to change this?  Can we just use slot 0/1?

Reply:
Yep! Reverted.